README.md

```diff
@@ -115,24 +102,35 @@ this can be generated using the command.
 This repository follows [Conventional Commits specification](https://www.conventionalcommits.org/en/v1.0.0/).
 
 ### Commit message format
-`<type>(<scope>): <subject>`  
 
-Samples:  
+`<type>(<scope>): <subject>`
+
+Samples:
+
 ```
 feat(login.strategy): add optional numBlocksBack config
 feat(deps): bump `passport-did-auth` and `iam-contracts` versions to latest
 ```
 
 ### Using CLI
-If you want to be lead step-by-step you can use `cz-cli` which will help you setup commit message according to our standards   
-`$ npm run commit`  
+
+If you want to be lead step-by-step you can use `cz-cli` which will help you setup commit message according to our standards  
+`$ npm run commit`
 
 In case if commit got interrupted (i.e. failed tests, failed eslint etc.) you can run  
 `$ npm run commit:retry`
 
 ### Troubleshooting
 
 `error: unknown option 'hook'`
-  
+
 In this case you need to initialize commitizen locally  
 `$ commitizen init cz-conventional-changelog --save-dev --save-exact`
+
+### Connection pooling
+
+To calculate maximum number of connections use following formula:  
+`{DBInstanceClassMemory/12582880}` - for example t2.micro has 512 MB RAM so maximum connections will be `(512*1024*1024)/12582880 ~= 40,`
+
+Then divide maximum connections number by number of maximum amount of IAM Cache Server instances.  
+If you are going to host 4 instances of ICS then `DB_MAXIMUM_CONNECTION_POOL` should be `40/4=10`
```

Comment:
Where is this formula derived from?

Reply:
Yeah, sory I did not notice that, will disable prettier for `md` files. For formula you need to ask @hejkerooo

Reply:
@hejkerooo could you comment on the rational for the formula please?

Reply:
Formula should be `LEAST({DBInstanceClassMemory/9531392}, 5000)` 

https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/CHAP_Limits.html

Reply:
Thanks @hejkerooo . What do you think of this change @Harasz ?
```suggestion
To calculate maximum number of connections use following formula:  
`LEAST({DBInstanceClassMemory/9531392}, 5000)`. This is taken from the [AWS RDS "Maximum number of database connections" documentation](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/CHAP_Limits.html).

Then divide maximum connections number by number of maximum amount of IAM Cache Server instances.  
If you are going to host 4 instances of ICS then `DB_MAXIMUM_CONNECTION_POOL` should be `40/4=10`
```